Makefile.common-jst

```diff
@@ -440,3 +440,29 @@ debug-printers: runtime-stdlib # required for $(ws_main) to work
 	@echo
 	@echo To load into ocamldebug, use:
 	@echo source \"$$(realpath _build/main/$(ocamldir)/tools/debug_printers)\"
+
+# boot-install installs the bootstrapped compiler into _install. It is mostly
+# just copied from the install phrase.
+# This allows tools/compare.sh to function as expected, which is useful because
+# it allows us to easily run the bootstrapped compiler, especially with
+# tools/compare.sh
+.PHONY: boot-install
```

Comment:
The target name `boot-install` is inconsistent with the target that installs the normal compiler into the `_install` directory, which is named `_install`.